src/modules/contacts/pages/ContactOverviewPage.vue

```diff
@@ -224,6 +226,8 @@ onBeforeMount(async () => {
     'contribution',
     'roles',
   ]);
+  contactAnnotations.notes = contact.value.profile.notes as string;
```

Comment:
I think this should be defaulted to empty string rather than casting `string | undefined` to `string`;
e.g.
```ts
contactAnnotation.notes = contact.value.profile.notes || '';
```